libraries/Microsoft.Bot.Connector/Authentication/AppCredentials.cs

```diff
@@ -234,42 +231,22 @@ protected virtual Lazy<IAuthenticator> BuildIAuthenticator()
                 LazyThreadSafetyMode.ExecutionAndPublication);
         }
 
+#pragma warning disable CA1801 // Review unused parameters
         private static bool IsTrustedUrl(Uri uri, ILogger logger)
+#pragma warning restore CA1801 // Review unused parameters
         {
-            lock (TrustedHostNames)
-            {
-                if (TrustedHostNames.TryGetValue(uri.Host, out var trustedServiceUrlExpiration))
-                {
-                    // check if the trusted service url is still valid
-                    if (trustedServiceUrlExpiration > DateTime.UtcNow.Subtract(TimeSpan.FromMinutes(5)))
-                    {
-                        return true;
-                    }
-
-                    logger.LogWarning($"{typeof(AppCredentials).FullName}.IsTrustedUrl(): '{uri}' found in TrustedHostNames but it expired (Expiration is set to: {trustedServiceUrlExpiration}, current time is {DateTime.UtcNow}).");
-                    return false;
-                }
-
-                logger.LogWarning($"{typeof(AppCredentials).FullName}.IsTrustedUrl(): '{uri}' not found in TrustedHostNames.");
-                return false;
-            }
+            return true;
```

Comment:
This method is now unused, it can be removed

Reply:
well spotted - next commit!